en-US/firefox-ios.xliff

```diff
@@ -1904,8 +1911,7 @@
       <trans-unit id="Settings">
         <source>Settings</source>
         <target>Settings</target>
-        <note>Accessibility label for the Settings button in the Tab Tray.
-   Settings</note>
+        <note>Settings</note>
```

Comment:
Delphine wrote: Not sure that "settings" as a localization note really explains the string "settings" here... would try to be clearer about where it is and what it refers to in this case (for ex. I've had a few localizers ask me in the past about "settings" string because they don't know if it refers to the browser or the system app, and they are sometimes translated differently)

Reply:
Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/3291